ucf.py

```diff
@@ -145,7 +145,9 @@ def save(self, filename=None):
         
         if self.rootfiles:
             container = _build_container(self.rootfiles)
-            self.meta[CONTAINER] = container.encode(UTF8)
+            if isinstance(container, unicode):
```

Comment:
`unicode` isn't a built-in in Python 3.x.

Reply:
Oh wow, I see I put a `unicode = str` compatibility thing in there, so that this will work fine with Py3. My bad.